Marlin/src/lcd/menu/menu_tramming.cpp

```diff
@@ -39,8 +39,10 @@
 //#define DEBUG_OUT 1
 #include "../../core/debug_out.h"
 
-float z_measured[G35_PROBE_COUNT] = { 0 };
+static float z_measured[G35_PROBE_COUNT];
+static bool z_isvalid[G35_PROBE_COUNT];
```

Comment:
My initial implementation initialized z_measured to NAN, but I found that the isnan() calls are **very** expensive. Using separate flag variables reduced FLASH usage by 38 bytes, although it did increase RAM usage by a few bytes.

If 3 bytes of RAM is more important than 38 bytes of FLASH, we could easily revert the commit where I made the switch.

Reply:
It's okay if `isnan()` is slow. It is surprising that it would cost 38 bytes more than adding an array and accompanying logic. According to my initial review of the code here, the array is initialized to `false` but I don't see anyplace where elements are being set to `true`.